.storybook/stories-from-readme.js

```diff
@@ -71,7 +72,7 @@ export function addPlaygroundStory(playgroundModule) {
 function AppProviderDecorator(story) {
   return (
     <div style={{padding: '8px'}}>
-      <Polaris.AppProvider>{story()}</Polaris.AppProvider>
+      <Polaris.AppProvider i18n={en}>{story()}</Polaris.AppProvider>
```

Comment:
It would be cool if we had a select to toggle between languages in dev

Reply:
addon-contexts should be able to let us do that https://www.npmjs.com/package/@storybook/addon-contexts

Reply:
Awesome 🎉